ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestinterfaces.py

```diff
@@ -71,22 +74,23 @@ def generate_test_details(interface: dict, description_template: str) -> tuple(s
         required_keys = ["name", "shutdown"]
 
         for interface_key, description_template in self.interface_types:
-            interfaces = get(self.structured_config, interface_key, [])
+            interfaces = self.structured_config.get(interface_key, [])
```

Comment:
it is the same but it helps readability
```suggestion
            interfaces = self.structured_config.get(interface_key, default=[])
```

Reply:
Added in 554e20b10cc2d157c35c92107939d8e54417c893